cmd/kured/main.go

```diff
@@ -129,6 +130,8 @@ func NewRootCommand() *cobra.Command {
 		"port number where metrics will listen")
 	rootCmd.PersistentFlags().IntVar(&drainGracePeriod, "drain-grace-period", -1,
 		"time in seconds given to each pod to terminate gracefully, if negative, the default value specified in the pod will be used")
+	rootCmd.PersistentFlags().StringVar(&drainPodSelector, "drain-pod-selector", "",
+		"label selector to filter pods on the node when draining (default: '')")
```

Comment:
Can we make it clear that this is an "inclusive" filter? Basically by using a pod-selector you are instructing drain to "delete only these pods that match this label selector".

Otherwise "filter" can suggest "filter out these pods", i.e., *don't* delete these pods.